doc/reference/storage_zfs.md

```diff
@@ -1,41 +1,42 @@
-(storage-zfs)=
+
 # ZFS - `zfs`
 
 ```{youtube} https://www.youtube.com/watch?v=ysLi_LYAs_M
 ```
 
-{abbr}`ZFS (Zettabyte file system)` combines both physical volume management and a file system.
-A ZFS installation can span across a series of storage devices and is very scalable, allowing you to add disks to expand the available space in the storage pool immediately.
+`ZFS (Zettabyte file system)` combines the roles of a file system and volume manager.
+A ZFS installation can span across multiple storage devices and is very scalable, allowing you to add disks to expand the available space in the storage pool immediately. It allows for multiple levels of redundancy of physical storage devices.
 
 ZFS is a block-based file system that protects against data corruption by using checksums to verify, confirm and correct every operation.
 To run at a sufficient speed, this mechanism requires a powerful environment with a lot of RAM.
 
 In addition, ZFS offers snapshots and replication, RAID management, copy-on-write clones, compression and other features.
 
-To use ZFS, make sure you have `zfsutils-linux` installed on your machine.
+To use ZFS on Ubuntu, make sure you have `zfsutils-linux` installed on your machine. Other distributions might use a different naming convention.
 
 ## Terminology
 
-ZFS creates logical units based on physical storage devices.
-These logical units are called *ZFS pools* or *zpools*.
-Each zpool is then divided into a number of *{spellexception}`datasets`*.
-These {spellexception}`datasets` can be of different types:
+ZFS creates logical units based on physical storage devices. These logical units are called *VDEVs* (virtual devices). VDEVs are made up of one or more physical disks, partitions, or even loop-mounted files; basically any block device. While a single disk can be used as a VDEV, most typically VDEVs are either 2 or more disks in a mirror configuration, 3 or more disks in the ZFS equivalent of a RAID5 (*raid-z1*), 4 or more disks configured as RAID6 (*raid-z2*), and so on. While the configuration of any particular VDEV is quite flexible, once created, VDEVs are immutable and cannot be changed (other than to replace a failed disk).
```

Comment:
"VDEV" and "VDEVs" must be added to the wordlist for the spellchecker (at `.sphinx/.wordlist.txt`).

Please spell out the numbers ("two" instead of "2" and so on).